packages/fluentui/react-northstar/src/components/Carousel/Carousel.tsx

```diff
@@ -155,13 +155,14 @@ class Carousel extends AutoControlledComponent<WithAsProp<CarouselProps>, Carous
 
   static defaultProps = {
     accessibility: carouselBehavior as Accessibility,
-    paddlePrevious: {},
-    paddleNext: {},
+    paddlePrevious: { previous: true },
+    paddleNext: { next: true },
```

Comment:
```suggestion
    paddlePrevious: {},
    paddleNext: {},
```

As I see we already have this in `defaultProps` in `.create()`